package.json

```diff
@@ -32,7 +32,11 @@
       "rollup": "catalog:tooling",
       "@babel/runtime": "catalog:tooling",
       "prismjs": "^1.30.0",
-      "typescript": "catalog:tooling"
+      "typescript": "catalog:tooling",
+      "react-aria": "catalog:react",
+      "react-aria-components": "catalog:react",
+      "react-stately": "catalog:react",
+      "@react-aria/interactions": "catalog:react"
```

Comment:
This is here to insure that we dont have any lurking dependency mismatches in our react-aria related dependency graph, since react-aria packages have dependencies on other react-aria packages (eg `react-stately` is a dep of both `react-aria` and `react-aria-components`